Index: newlib-snapshot/src/libgloss/arm/elf-nano.specs
===================================================================
--- newlib-snapshot.orig/src/libgloss/arm/elf-nano.specs
+++ newlib-snapshot/src/libgloss/arm/elf-nano.specs
@@ -3,7 +3,7 @@
 %rename cpp_unique_options		nano_cpp_unique_options
 
 *cpp_unique_options:
--isystem =/include/newlib-nano %(nano_cpp_unique_options)
+-isystem /usr/include/newlib/nano %(nano_cpp_unique_options)
 
 *nano_libc:
 -lc_nano
