Index: newlib-snapshot/src/newlib/Makefile.am
===================================================================
--- newlib-snapshot.orig/src/newlib/Makefile.am
+++ newlib-snapshot/src/newlib/Makefile.am
@@ -165,6 +165,7 @@ libc.a: libc/libc.a libm.a
 	rm -rf libc.a libg.a tmp
 	mkdir tmp
 	cd tmp; \
+	LC_ALL=C ; \
 	 $(AR) x ../libm.a $(MATHOBJS_IN_LIBC) ; \
 	 $(AR) x ../libc/libc.a ; \
 	 $(AR) $(AR_FLAGS) ../$@ *.o
Index: newlib-snapshot/src/newlib/Makefile.in
===================================================================
--- newlib-snapshot.orig/src/newlib/Makefile.in
+++ newlib-snapshot/src/newlib/Makefile.in
@@ -921,6 +921,7 @@ uninstall-am: uninstall-toollibDATA unin
 @USE_LIBTOOL_FALSE@	rm -rf libc.a libg.a tmp
 @USE_LIBTOOL_FALSE@	mkdir tmp
 @USE_LIBTOOL_FALSE@	cd tmp; \
+@USE_LIBTOOL_FALSE@	LC_ALL=C; \
 @USE_LIBTOOL_FALSE@	 $(AR) x ../libm.a $(MATHOBJS_IN_LIBC) ; \
 @USE_LIBTOOL_FALSE@	 $(AR) x ../libc/libc.a ; \
 @USE_LIBTOOL_FALSE@	 $(AR) $(AR_FLAGS) ../$@ *.o
Index: newlib-snapshot/src/newlib/libc/Makefile.am
===================================================================
--- newlib-snapshot.orig/src/newlib/libc/Makefile.am
+++ newlib-snapshot/src/newlib/libc/Makefile.am
@@ -109,6 +109,7 @@ libc.a: $(SUBLIBS)
 	rm -rf tmp
 	mkdir tmp
 	cd tmp; \
+	LC_ALL=C ; \
 	 for i in $(SUBLIBS); do \
 	   $(AR) x ../$$i; \
 	 done; \
Index: newlib-snapshot/src/newlib/libc/Makefile.in
===================================================================
--- newlib-snapshot.orig/src/newlib/libc/Makefile.in
+++ newlib-snapshot/src/newlib/libc/Makefile.in
@@ -1032,6 +1032,7 @@ uninstall-am: uninstall-dvi-am uninstall
 @USE_LIBTOOL_FALSE@	rm -rf tmp
 @USE_LIBTOOL_FALSE@	mkdir tmp
 @USE_LIBTOOL_FALSE@	cd tmp; \
+@USE_LIBTOOL_FALSE@	LC_ALL=C; \
 @USE_LIBTOOL_FALSE@	 for i in $(SUBLIBS); do \
 @USE_LIBTOOL_FALSE@	   $(AR) x ../$$i; \
 @USE_LIBTOOL_FALSE@	 done; \
Index: newlib-snapshot/src/newlib/libc/sys/linux/Makefile.am
===================================================================
--- newlib-snapshot.orig/src/newlib/libc/sys/linux/Makefile.am
+++ newlib-snapshot/src/newlib/libc/sys/linux/Makefile.am
@@ -196,6 +196,7 @@ lib.a:  $(lib_a_OBJECTS)
 	rm -rf tmp
 	mkdir tmp
 	cd tmp; \
+	LC_ALL=C ; \
 	for i in $(SUBLIBS); do \
 	  $(AR) x ../$$i; \
 	done; \
Index: newlib-snapshot/src/newlib/libc/sys/linux/Makefile.in
===================================================================
--- newlib-snapshot.orig/src/newlib/libc/sys/linux/Makefile.in
+++ newlib-snapshot/src/newlib/libc/sys/linux/Makefile.in
@@ -1599,6 +1599,7 @@ uninstall-am:
 @USE_LIBTOOL_FALSE@	rm -rf tmp
 @USE_LIBTOOL_FALSE@	mkdir tmp
 @USE_LIBTOOL_FALSE@	cd tmp; \
+@USE_LIBTOOL_FALSE@	LC_ALL=C; \
 @USE_LIBTOOL_FALSE@	for i in $(SUBLIBS); do \
 @USE_LIBTOOL_FALSE@	  $(AR) x ../$$i; \
 @USE_LIBTOOL_FALSE@	done; \
Index: newlib-snapshot/src/newlib/libm/Makefile.am
===================================================================
--- newlib-snapshot.orig/src/newlib/libm/Makefile.am
+++ newlib-snapshot/src/newlib/libm/Makefile.am
@@ -25,6 +25,7 @@ libm.a: $(SUBLIBS)
 	rm -rf tmp
 	mkdir tmp
 	cd tmp; \
+	LC_ALL=C ; \
 	  for i in $(SUBLIBS); do \
 	    $(AR) x ../$$i; \
 	done; \
Index: newlib-snapshot/src/newlib/libm/Makefile.in
===================================================================
--- newlib-snapshot.orig/src/newlib/libm/Makefile.in
+++ newlib-snapshot/src/newlib/libm/Makefile.in
@@ -898,6 +898,7 @@ uninstall-am: uninstall-dvi-am uninstall
 @USE_LIBTOOL_FALSE@	rm -rf tmp
 @USE_LIBTOOL_FALSE@	mkdir tmp
 @USE_LIBTOOL_FALSE@	cd tmp; \
+@USE_LIBTOOL_FALSE@	LC_ALL=C; \
 @USE_LIBTOOL_FALSE@	  for i in $(SUBLIBS); do \
 @USE_LIBTOOL_FALSE@	    $(AR) x ../$$i; \
 @USE_LIBTOOL_FALSE@	done; \
